Development

Alexandru-Bulai opened the pull request.

Alexandru-Bulai committed:
First commit, basic structure of the project

Alexandru-Bulai committed:
separate CSS style from index.html

Alexandru-Bulai committed:
minor correction on README file

Alexandru-Bulai committed:
small changes on index.html no impact on project

Alexandru-Bulai committed:
lots of changes on CSS structure and index.html

Alexandru-Bulai committed:
remove "lang-en" from html opening tags

Alexandru-Bulai committed:
add content to index.html, small changes on CSS

Alexandru-Bulai committed:
change from <ul> element to <nav> element

Alexandru-Bulai committed:
adjuste layout of menu to be more centered

Alexandru-Bulai committed:
change company logo, header style and page scaling

Alexandru-Bulai committed:
change introduction section position

Alexandru-Bulai committed:
change from <'ol'> element to grid in home.html

vladiiancu commented:
Nice improvements on the commit messages.

vladiiancu commented:
A good PR will also have a short summary of the changes, the reasons etc. It is a bit harder for the first few ones but it becomes easier. 
A screenshot with how the page looks like now after all the changes is also a nice addition

vladiiancu commented on how_to.html:
I would not include an empty page in a PR.

vladiiancu commented on index.html:
While your change from index.html to home.html is fine, you should know this: Most web servers, by default are looking at the root directory for one of the following files: index.html, index.htm, index.php and so on. 

Of course, the settings can be changed without a problem but that's why, probably most of the websites have an index.html as the main file. 

You can keep the change to home.html - no worries. Just some background info for you.

vladiiancu commented on CSS/style.css:
```diff
@@ -0,0 +1,121 @@
+}
+
+.home_title {
+  font-style: italic;
```
font-style: italic on a class used on a p element. And that p element, contains a `<strong>` element. So, would it make more sense to have 
`<em><strong>Bold and italic text</strong> and some more italic text</em>`
And have this wrapped inside a `<p>`.

instead of
`<p class="home_title"><strong>Bold and italic text</strong> and some more italic text</p>`   ?

vladiiancu commented on home.html:
```diff
@@ -0,0 +1,63 @@
+            Some services offer pick-up and drop-off, and you can easily find sitters in your neighborhood. Take advantage of additional services such as grooming and veterinary appointments that can be coordinated with your pet's stay, saving you time and effort.
+        </div>
+        <div class="grid_item">
+            <p class="home_grid_item"><strong>Peace of Mind</strong></p>
```
When you reach the CSS lectures, make note of the BEM naming convention https://en.bem.info/methodology/naming-convention/ 
It might come in handy :)

vladiiancu commented on Assets/images/home/gif1-500x.gif:
Nice :+1:

Alexandru-Bulai commented:
How would you do it? 
Should I create a new folder just for that and in each commit I put the PR there?

Alexandru-Bulai committed:
add images to grid container

Alexandru-Bulai reviewed (commented):
PR  7-12-23:

Removed the "p" elements from the grid and replaced them with a "div" instead for better manipulation object.
Added images on each div on the grid that represent  the descipriton.

Changed the background-color of the body for better UI 

Screeshot of the home page with changes applied: 
![screenshot page 7-12](https://github.com/Alexandru-Bulai/website-project/assets/114404620/1dffb78e-4573-4dc5-9e3e-74805ab48909)

vladiiancu reviewed (changes requested):
Please make changes according to the leftover comments or, if no changes made now, acknowledge that you read the comment so that we can continue

Alexandru-Bulai committed:
add footer to the page

Alexandru-Bulai committed:
add footnote and realigne header menu

vladiiancu commented on home.html:
```diff
@@ -0,0 +1,125 @@
+        </div>
+    </div>
+    <div class="footer__socialMedia">
+        <img src="Assets/images/footer/facebook.png" alt="Facebook icon">
```
don't you need to have links on each one of them?

vladiiancu commented on home.html:
```diff
@@ -0,0 +1,125 @@
+    </div>
+
+    <div class="home_image">
+        <img src="Assets/images/home/gif1-500x.gif" alt="GIF">
```
alt text should be more descriptive

Alexandru-Bulai commented:
PR 12-12-23  13:00 :

Footer : 
changed structure into a more profesisonal looking footer, added social media imgs and location button

Header : 
changed menu style and hover effect
 
![Screenshot 2023-12-12 130128](https://github.com/Alexandru-Bulai/website-project/assets/114404620/c7ac29f4-a9b8-49e7-9fa4-c0e3cb9e0333)
![Screenshot 2023-12-12 130155](https://github.com/Alexandru-Bulai/website-project/assets/114404620/5a2f9e32-9993-485e-b1e6-f55e6643c35f)

Alexandru-Bulai committed:
final design footer

Alexandru-Bulai commented:
PR 12-12-23 15:30 :

Footer :

finished design on footer and added links on social media logos.

![Screenshot 2023-12-12 153752](https://github.com/Alexandru-Bulai/website-project/assets/114404620/88a60c94-2662-4a10-99e5-d98965e57f2c)
